spec/components/interview_preferences_component_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe InterviewPreferencesComponent do
+  it 'renders `Not answered yet` if `#interview_preferences` is nil' do
+    application_form = instance_double(
+      ApplicationForm,
+      interview_preferences: nil,
+    )
+    result = render_inline(described_class.new(application_form: application_form))
+    expect(result.text).to include('Not answered yet')
```

Comment:
I think "yet" implies they might answer it in the future, but they won't be able to. How about "No preferences"?